Soccar [Library]/src/nl/soccar/library/Game.java

```diff
@@ -80,9 +75,32 @@ public void stop() {
             return;
         }
 
+        timeLeft = 0;
         status = GameStatus.STOPPED;
+    }
+
+    public void decreaseGameTime() {

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)